Source/Mothership/TMoth.cpp

```diff
@@ -727,33 +793,33 @@ void TMoth::OnIdle()
 {
     // queues may be changing but we can deal with a static snapshot of the actual
     // queue because OnIdle will execute periodically.
-    WALKMAP(uint32_t,deque<P_Msg_t>*,TwigExtMap,D)

```

Comment:
Hooray for NS integration! As far as I'm concerned, you can outright delete these lines (assuming you were keeping them around for my benefit).

Reply:
They were being kept around for the benefit of whoever was going to end up re-writing Mothership - externals were never tested, so I removed all code that tried to deal with them.